platforms/common/src/CommonKernels.cpp

```diff
@@ -4994,27 +5040,28 @@ void CommonCalcGayBerneForceKernel::initialize(const System& system, const GayBe
     exceptionParams.initialize<mm_float2>(cc, max(1, numExceptions), "exceptionParams");
     if (numExceptions > 0)
         exceptionParams.upload(exceptionParamsVec);
-    
+
     // Create data structures used for the neighbor list.
 
-    int numAtomBlocks = (numRealParticles+31)/32;
+    const int tileSize = cc.getTileSize();
```

Comment:
Another one that shouldn't depend on the tile size used by the main nonbonded kernel.